### PR TITLE
152 - Add Opera (+ Edge)

### PR DESCRIPTION
**Pull request by KMariusz**
Added Opera & Edge support

### CODE REVIEW

**Comment by KMariusz** [comment]
Can't run tests on CLI - there are no Opera and Edge on grid.
Second issue are failing tests from selenium module - this should start working after fixes from other task (fixed by M.Dzienia)

Local run of 58 tests:
Opera -  37 Fail (testGetDriver Passed)
Firefox - 36 Fail
Chrome - 40 Fail

So it gives similiar results :)

Edge
I have no Win 10 to test if it is working for sure. 
I've used Edge Dev for Windows 7 + some additional lines of code (which are not included in pull req) special for that version and it was working fine (same as Opera/Chrome/Firefox)

**Comment by KMariusz** [comment]
WebDriver for Microsoft Edge (EdgeHTML) - the one build into Win 10 OS by default - is Feature On Demand from the latest version (18).
https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/#downloads

My solution is another parameter with flag to tell if WebDriver was already installed as FoD.
If it is then automatic download is not triggered because it downloads driver up to version 17.
I leave "standard" solution to support older versions.

Tested on VM with Latest Win 10 build and it works locally fine.

**Review by tompalka — APPROVED** [review]

File: mrchecker-framework-modules/mrchecker-selenium-module/src/test/java/com/capgemini/mrchecker/selenium/core/base/properties/PropertiesManagerTest.java [hunk 1]
```diff
@@ -1,68 +1,72 @@
...
-import org.junit.AfterClass;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Test;
-
-import com.capgemini.mrchecker.selenium.core.base.properties.PropertiesSelenium;
 import com.capgemini.mrchecker.test.core.base.properties.PropertiesSettingsModule;
 import com.google.inject.Guice;
 import com.google.inject.Injector;
+import org.junit.*;
```
Comment (przwojtkow) [inline mrchecker-framework-modules/mrchecker-selenium-module/src/test/java/com/capgemini/mrchecker/selenium/core/base/properties/PropertiesManagerTest.java:6]: Do we want to import everything from that package?

**Review by przwojtkow — APPROVED** [review]
Great job! Please find one remark from my end. If you feel comfortable, please feel free to merge.
I have one suggestion about overall process of PR. Next time please try to reduce number of files changes within single PR. It will be easier to check. If it is needed please do not hesitate to split the tasks from our board into smaller units.

**Pull request merged**